File: 5622.c
#include <stdio.h>
#include <string.h>

int main(void)
{
	char input[20];
	int num = 0;
	scanf("%s", input);
	
	for(int i = 0; i < strlen(input); ++i) {
		if(input[i] >= 65 && input[i] <=67) /*abc*/{
			num += 3;
		}
		if(input[i] >= 68 && input[i] <=70 /*def*/) {
			num += 4;
		}
		if(input[i] >= 71 && input[i]	<= 73/*ghi*/) {
			num += 5;
		}
		if(input[i] >= 74 && input[i] <= 76 /*jkl*/) {
			num += 6;
		}
		if(input[i] >= 77 && input[i] <= 79 /*mno*/) {
			num += 7;
		}
		if(input[i] >= 80 && input[i] <= 83 /*pqrs*/) {
			num += 8;
		}
		if(input[i] >=84 && input[i] <= 86 /* tuv*/) {
			num += 9;
		}
		if(input[i] >= 87 && input[i] <= 90 /*wxyz*/) {
			num += 10;
		}
	}
	printf("%d\n", num);
	return 0;
}

File: 5622.py
INPUT = list(input())
num = 0
for i in range (len(INPUT)):
	if ord(INPUT[i]) >= 65 and ord(INPUT[i]) <= 67:
			num += 3
	if ord(INPUT[i]) >= 68 and ord(INPUT[i]) <= 70:
			num += 4	
	if ord(INPUT[i]) >= 71 and ord(INPUT[i]) <= 73:
			num += 5
	if ord(INPUT[i]) >= 74 and ord(INPUT[i]) <= 76: 
			num += 6
	if ord(INPUT[i]) >= 77 and ord(INPUT[i]) <= 79:
			num += 7
	if ord(INPUT[i]) >= 80 and ord(INPUT[i]) <= 83:
			num += 8
	if ord(INPUT[i]) >= 84 and ord(INPUT[i]) <= 86:
			num += 9
	if ord(INPUT[i]) >= 87 and ord(INPUT[i]) <= 90:
			num += 10
		
print(num)
